src/main/java/org/openstreetmap/atlas/proto/adapters/ProtoPackedTagStoreAdapter.java

```diff
@@ -114,6 +114,7 @@ public ProtoSerializable deserialize(final byte[] byteArray)
         Field indexField = null;
         Long index = -1L;
 
+        // Consider using getters, says nick
```

Comment:
This is just a note, should we do this in another PR?

Reply:
"says Nick" :) :)